functionalDB/chapter.md

```diff
@@ -1,103 +1,104 @@
-# Designing a database like an archaeologist
+<!-- American spelling --> 
+# CircleDB: An Archaeology-Inspired Database
 
-Software development is often viewed as a rigorous process, where the inputs are requirements and the output is the working product. However, software developers are people with their own perspectives and biases which color the outcome of their work. 
+Software development is often viewed as a rigorous process, where the inputs are requirements and the output is the working product. However, software developers are people, with their own perspectives and biases which color the outcome of their work. 
 
-In this chapter, we will explore how a change in a common perspective affects the design and implementation of a well-studied type of software -- a database.
+In this chapter, we will explore how a change in a common perspective affects the design and implementation of a well-studied type of software: a database.
 
 ## Introduction 
 
-Database systems are designed to store and query data. This is something that all information workers do; however, the systems themselves were designed by computer scientists. As a result, modern database systems are highly influenced by what a computer scientist’s definition of data is, and what can be done with it. 
+Database systems are designed to store and query data. This is something that all information workers do; however, the systems themselves were designed by computer scientists. As a result, modern database systems are highly influenced by computer scientists’ definition of what data is, and what can be done with it. 
 
-For example, most modern databases implement updates by overwriting old data in-place instead of appending the new data and keeping the old. This mechanism, nicknamed "place oriented programming" by Rich Hickey [REF], saves storage space but makes it impossible to retrieve the entire history of a particular record. This design decision reflects the computer scientist’s perspective that ‘history’ is less important than the price of its storage. 
+For example, most modern databases implement updates by overwriting old data in-place instead of appending the new data and keeping the old. This mechanism, nicknamed "place-oriented programming" by Rich Hickey [REF], saves storage space but makes it impossible to retrieve the entire history of a particular record. This design decision reflects the computer scientist’s perspective that "history" is less important than the price of its storage. 
 
-If you were to instead ask an archaeologist what should be done with the old data, the answer would be "hopefully, it’s just buried underneath".
+If you were to instead ask an archaeologist what should be done with the old data, the answer would be "hopefully, just bury it underneath".

```

Comment:
Actually, the question to the archaeologist should have been "where can we find the old data", and not what to do with it